static/js/debug.js

```diff
@@ -131,3 +131,27 @@ IterationProfiler.prototype = {
         }
     },
 };
+
+export function test_send_many_messages(stream, subject, count) {
+    var num_sent = 0;
+
+    function do_send_one() {
+        var message = {};
+        num_sent += 1;
+
+        message.type = "stream";
+        message.to = stream;
+        message.subject = subject;
+        message.content = num_sent.toString();
+
+        compose.send_message(message);
```

Comment:
You have to export `send_message` in `compose.js for this to work.  I think it's fine to just do something like this in `compose.js`:

~~~ js
exports.send_test_message = send_message
~~~

That way we're not exposing send_message directly to the outside world, and it would be a smaller diff.  You'll want to test this manually, obviously.